### PR TITLE
Shultsedu 286 localization

### PR DESCRIPTION
**Pull request by nadiiazhurba**

### CODE REVIEW

**Review by YuliaLoyko — CHANGES_REQUESTED** [review]
i actually thought that we're going to implement localization, which is not just moving strings to resx files, but provide the mechanism of translation?

File: SideMenuFlyoutShellTestProject.iOS/Info.plist [hunk 1]
```diff
@@ -2,37 +2,39 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-    <key>UIDeviceFamily</key>
```
[the remark this answers is not in the document]
Comment (nadiiazhurba) [reply SideMenuFlyoutShellTestProject.iOS/Info.plist:5]: It is the issue with it that appears after opening in different IDEs. I'm trying not to change it on purpose.

**Comment by nadiiazhurba** [comment]
> actually thought that we're going to implement localization, which is not just moving strings to resx files, but provide the mechanism of translation?

Actually, from now on localization should work by itself. If I add more files with another languages and user have different language settings he should see app in it's preferred local. And also with each new resx file I should add language in info.plist.

**Comment by nadiiazhurba** [comment]
I believe that we haven't discussed earlier such changes as a mechanism of language changes if you mean something like new Settings page with possibilities to switch between different locales. Only adding strings to resources. Which, actually means adding possibility to extend application with new locales.

File: SideMenuFlyoutShellTestProject.iOS/Info.plist (hunk 1 above)
[the remark this answers is not in the document]
Comment (YuliaLoyko) [reply SideMenuFlyoutShellTestProject.iOS/Info.plist:5]: so don't commit if no actual change)

**Comment by YuliaLoyko** [comment]
> > actually thought that we're going to implement localization, which is not just moving strings to resx files, but provide the mechanism of translation?
> 
> Actually, from now on localization should work by itself. If I add more files with another languages and user have different language settings he should see app in it's preferred local. And also with each new resx file I should add language in info.plist.

so let's add another files for different locales) you can translate just a couple of strings - to show that it works

File: SideMenuFlyoutShellTestProject.iOS/Info.plist (hunk 1 above)
[the remark this answers is not in the document]
Comment (nadiiazhurba) [reply SideMenuFlyoutShellTestProject.iOS/Info.plist:5]: There was an actual change. I've added preferred locale for an iOS app, as it was suggested in tutorial. Lines 37-38. I'm sorry that I haven't written it earlier, so you wouldn't have to check the difference by yourself.

<key>CFBundleDevelopmentRegion</key>
	<string>en</string>

**Comment by nadiiazhurba** [comment]
> > > actually thought that we're going to implement localization, which is not just moving strings to resx files, but provide the mechanism of translation?
> > 
> > 
> > Actually, from now on localization should work by itself. If I add more files with another languages and user have different language settings he should see app in it's preferred local. And also with each new resx file I should add language in info.plist.
> 
> so let's add another files for different locales) you can translate just a couple of strings - to show that it works

Sure, no problem. I'll do it.

File: SideMenuFlyoutShellTestProject.iOS/Info.plist (hunk 1 above)
[the remark this answers is not in the document]
Comment (YuliaLoyko) [reply SideMenuFlyoutShellTestProject.iOS/Info.plist:5]: so commit only actual change)

**Comment by nadiiazhurba** [comment]
> > > > actually thought that we're going to implement localization, which is not just moving strings to resx files, but provide the mechanism of translation?
> > > 
> > > 
> > > Actually, from now on localization should work by itself. If I add more files with another languages and user have different language settings he should see app in it's preferred local. And also with each new resx file I should add language in info.plist.
> > 
> > 
> > so let's add another files for different locales) you can translate just a couple of strings - to show that it works
> 
> Sure, no problem. I'll do it.

Done it

**Review by YuliaLoyko — APPROVED** [review]

**Pull request merged**